.gitignore

```diff
@@ -1,4 +1,4 @@
-node_modules
+node_modules/
```

Comment:
Actually about adding `node_modules` to .gitignore we had already thought of this, but there is some requirement for github actions to keep the node_modules folder so that it can run the action in other repositories, check out #20 for more details into this, as of now can you please keep the node_modules?